Cargo.lock

```diff
@@ -225,11 +234,11 @@ version = "0.2.3"
 dependencies = [
  "itertools 0.14.0",
  "partial-min-max",
- "rand",
+ "rand 0.9.0",
```

Comment:
Issue: Why do we now have two versions of `rand`

Reply:
phf_generator needs to be updated upstream